cmds/contest/server/server.go

```diff
@@ -221,6 +231,14 @@ func ServerMain(cmd string, args []string, sigs <-chan os.Signal) error {
 
 	err = jm.Run(ctx, *flagResumeJobs)
 
+	target.SetLocker(nil)
+	if err := storage.SetStorage(nil); err != nil {
+		log.Errorf("Failed to flush storage")
+	}
+	if err := storage.SetAsyncStorage(nil); err != nil {
+		log.Errorf("Failed to flush async storage")
+	}
```

Comment:
From how I read this, this will never error with `nil` as a storage engine

Reply:
yes, and this is fine: if user want to disable storage, they should be allowed to. we do it when tearing down tests.

Reply:
Yeah I get the disable usecase, but not sure what we are printing here, as the log lines can never be reached?

Reply:
oh yeah, but lint will complain if i don't check the return value :)
but i guess i can simplify it by using a sentinel